### PR TITLE
Rename Commute > AreCommutingNonProjectiveMatrices

### PR DESCRIPTION
**Pull request by ssiccha**

### CODE REVIEW

File: gap/projective/AnSnOnFDPM.gi [hunk 1]
```diff
@@ -58,7 +58,7 @@ end;
 
 ## Functions for efficient matrix operations
 
-RECOG.Commute := function(g, h);
+RECOG.AreCommutingNonProjectiveMatrices := function(g, h);
```
Comment (fingolfin) [inline gap/projective/AnSnOnFDPM.gi:61]: Matrices are by default not projective, so this feels overly redundant.
```suggestion
RECOG.AreCommutingMatrices := function(g, h);
```
Also, one immediately wonders why this doesn't use your new AreCommuting/DoCommute methods. Presumably there is a reason -- if so, it should be stated in a comment. (My guess is: you are concerned this is called with "projective" groups)

Finally, why change this but not the Conj and ConjInv methods afterwards?

Comment (FriedrichRober) [reply gap/projective/AnSnOnFDPM.gi:61]: I agree. It is unclear why we shouldn't use the new `AreCommuting/DoCommute` method. I thought the idea was to use this the same way as the `isone` entry in the recog node in order to deal with projective groups.

**Comment by ssiccha** [comment]
I'll try to remove the `ConvertToMatrixRep` calls. To make sure this does not slow down thinkgs, I'll check with asserts whether the input is in the correct representation.

**Comment by ssiccha** [comment]
Actually, I have thought about this again, see #260. I'm closing this PR since the whole file might be deleted anyways.

**Pull request closed without merging**